docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+  
+  
+</details>
+
+
+<details>
+  <summary>What server-side technologies does Amplication use?</summary>
+  
+NestJS, Prisma, PostgreSQL, Passport, GraphQL, Swagger UI, Jest, Docker
+  
+</details>
+
+
+<details>
+  <summary>What client-side technologies does Amplication use?</summary>
+  
+ReactJS, React-Admin, Axios, Formik 
+  
+</details>
+
+
+<details>
+  <summary>Is Amplication a no-code solution?</summary>
+  
+Amplication is a tool for professional developers, enabling you to automate repetitive tasks, eliminating or reducing the time spent on repetitive server-side tasks and boilerplate code. However, your developer skills are very much required to create your  hand-crafted business logic, so it is definitely NOT a no-code solution. Some would call Amplication a low-code solution because it reduces the amount of code you need to write. 
+  
+</details>
+
+
+<details>
+  <summary>What is the Amplication Architecture?</summary>
+  
+The generated app is built from two projects, each in a separate folder:
+
+- Server - For all the server components including REST API, GraphQL, Services and more.
+- Admin - For the Admin UI including forms for CRUD operations on all data models.
+  
+</details>
+
+
+
+
+<details>
+  <summary>Can I customize the code generated by Amplication?</summary>
+  
+Definitely. The code is yours. Do with it what you wish. 
+  
+</details>
+
+
+
+<details>
+  <summary>Can I contribute to the development of Amplication?</summary>
+  
+We'd love to have you contribute! To find out how, read:
+
+[Contributing](https://docs.amplication.com/docs/contributing/)
+
+  
+</details>
+
+
+
+
+<details>
+  <summary>Can I work with GitLab (or other git providers) besides GitHub?</summary>
+  
+Amplication currently supports integration only with GitHub. 
+  
+</details>
+
+
+
+<details>
+  <summary>Which databases does Amplication support?</summary>
+  
+Amplication currently supports only PostgreSQL.
+However, we generate apps that work with Prisma, so you could change the Prisma configuration to use another database supported by Prisma.  
+  
+</details>
+
+
+
+<details>
+  <summary>Which Cloud Platforms does Amplication Support?</summary>
+
+You can host your server on AWS, Heroku, Azure, Google Cloud Platform, Digital Ocean, many and more. 
+  
+Amplication uses Docker and Kubernetes to support deployment. 
+
+Check out the following for for more information: 
+
+[Deploy a Docker Container](https://docs.amplication.com/docs/deploy/#deploy-a-docker-container)
+
+[Deploy Amplication to a Kubernetes cluster using Helm](https://docs.amplication.com/docs/helm-chart/)
+
+[Deploying Amplication Server to Digital Ocean](https://dev.to/asiancat54x/deploying-amplication-app-to-digital-ocean-5d7k)
+
+  
+</details>
+
+
+
+
+
+<details>
+  <summary>Is Amplication Self-Hosted? </summary>
```

Comment:
<summary>Is Amplication self-hosted? </summary>

Reply:
Removed this section